data/classificator.txt

```diff
@@ -222,6 +222,7 @@ world  +
         beef_bowl  -
         brazilian  -
         breakfast  -
+        bubble_tea  -
```

Comment:
@vng how did we miss that? The currently generated data won't have bubble tea, right?

Reply:
we need to add validation to data files changes